Leftover vertical org changes (#233)
wasp-lang/open-saas: opensaas-sh/blog/src/content/docs/start/guided-tour.md

`e2e-tests` contains the end-to-end tests using Playwright, which you can run to test your app's functionality.

### App File Structure

> We've structured this full-stack app template according to feature. That means that most directories within `app/src` contain both the React client code and NodeJS server code necessary for implementing its logic. 

Reviewer: ```suggestion
We've structured this full-stack app template vertically (by feature). That means that most directories within `app/src` contain both the React client code and NodeJS server code necessary for implementing its logic. 
```

It's possible to learn Wasp's feature set simply through using this template, but if you find yourself unsure how to implement a Wasp-specific feature and/or just want to learn more, a great starting point is the intro tutorial in the [Wasp docs](https://wasp-lang.dev/docs) which takes ~20 minutes.
:::

> ### Client

Reviewer: what about this stuff, isnt there some stuff still left in clinet and server that we want to explain here?

Author: `├── client/            # Shared components, hooks, landing page, and other client code (React).`
`├── server/            # Scripts, shared server utils, and other server-specific code (NodeJS).`

Author: Ok I added those sections back and revised them a bit :)
